projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.html

```diff
@@ -68,7 +68,7 @@
     <header class="igx-excel-filter__menu-header">
         <h4>{{ column.header || column.field }}</h4>
         <div *ngIf="grid.displayDensity!=='comfortable'" class="igx-excel-filter__menu-header-actions">
-            <button *ngIf="column.selectable" igxButton="icon"
+            <button *ngIf="grid.columnSelection !== 'none' && column.selectable" igxButton="icon"
```

Comment:
Instead of using string values for the default mode when checking whether to add "Select column" item in the filtering dialog , wouldn't it be better to use a getter for the default GridSelectionMode. Not to mention that if one day we change the default mode to 'multple' it will be required only one change instead of replacing strings in the `Excel filtering template`

Example:
```
private _defaultColumnSelectionMode = GridSelectionMode.none;

    /**
     * Returns the current column.
     */
    public get defaultColumnSelectionMode(): GridSelectionMode {
        return this._defaultColumnSelectionMode;
    }
```

`<button *ngIf="grid.columnSelection !== defaultColumnSelectionMode ..`